tests/suite_utils.py

```
@@ -136,12 +138,26 @@ def stress_oom(
     Keep writing until we hit OOM.
     """
     logging.info("Starting stress test to trigger OOM with query size '%s'", query_size)
+    cluster_endpoint , port = instance.get_cluster_endpoint()['endpoint'], instance.get_cluster_endpoint()['port']
     db = instance.create_connection(ssl=ssl, network_type=network_type)
-    g = db.select_graph("test")
     big = "UNWIND RANGE(1, 100000) AS id CREATE (n:Person {name: 'Alice'})"
     medium = "UNWIND RANGE(1, 25000) AS id CREATE (n:Person {name: 'Alice'})"
     small = "UNWIND RANGE(1, 10000) AS id CREATE (n:Person {name: 'Alice'})"
+    scheme = "rediss" if ssl else "redis"
+    url = f'{scheme}://falkordb:{instance.falkordb_password}@{cluster_endpoint}:{port}'
```

Ruff (0.12.2)

Local variable `url` is assigned to but never used

Remove assignment to unused variable `url`

(F841)